groovy/postInit/chemistry/organic_chemistry/Solvents.groovy

```diff
@@ -67,6 +68,7 @@ CSTR.recipeBuilder()
 // TCE (trichloroethylene)
 
 CSTR.recipeBuilder()
+        .circuitMeta(2)
         .notConsumable(ore('springKanthal'))
```

Comment:
if you need this spring, just move this recipe to a batch reactor and multiply values by 20. otherwise, you can just remove it.

Reply:
done